### ISSUE TITLE
About nSpeakers setting

### ISSUE THREAD

**Issue by Erunning** [clovaai/voxceleb_trainer#18]
Hi，thanks for your sharing!
I have a question.
When training example, should I set nSpeakers as 5594 ? Because the number of speakers in VOXCELEB2 DEV is 5994. I noticed the defaulted nSpeakers is 6200.

`parser.add_argument('--nSpeakers', type=int, default=6200,  help='Number of speakers in the softmax layer for softmax-based losses, utterances per speaker per iteration for other losses');`

`python ./trainSpeakerNet.py --model ResNetSE34 --encoder SAP --trainfunc amsoftmax --optimizer adam --save_path data/exp1 --batch_size 200 --max_frames 200 --scale 30 --margin 0.3 --train_list /home/joon/voxceleb/train_list.txt --test_list /home/joon/voxceleb/test_list.txt --train_path /home/joon/voxceleb/voxceleb2 --test_path /home/joon/voxceleb/voxceleb1`

**Comment by joonson** [comment]
Yes, for softmax 5994 would be ideal, although a larger number probably won't affect performance.
